### PR TITLE
more documentation

### PR DESCRIPTION
**Pull request by dollschasingmen**

### CODE REVIEW

**Comment by dollschasingmen** [comment]
@rfroetscher

File: docs/ADVANCED_CONFIGURATION.md [hunk 1]
```diff
@@ -1,18 +1,80 @@
...
+*File* - [redshift.yml](../config/example/redshift.yml)
+
+*Properties*
+- Rails env
+  - host
+  - database
+  - port
 
 ## Github
+Connection details to git are in [environment variables](ENVIRONMENT_VARIABLES.md). This only configures which branch your query versions are maintained on. Configuring git is optional but highly recommended.
```
Comment (rfroetscher) [inline docs/ADVANCED_CONFIGURATION.md]: github instead of git here

File: docs/ADVANCED_CONFIGURATION.md [hunk 2]
```diff
@@ -1,18 +1,80 @@
...
+- database - this means you will use [RailsAdmin](https://github.com/sferik/rails_admin) to administer user authorization
+- disabled - this means anyone can log in as a Guest with role Admin. Use this for development settings ONLY.
+
+*File* - [config.yml](../config/example/role_hierarchy.yml)
+
+*Properties*
+- auth_type (`saml`/`database`/`disabled`)
+
+#### Database (RailsAdmin) Authorization
+One quirk: you need to have auth_type set to `disabled` to create the first user with role *Admin*. Then you can change to `database` and use that first user to create other users and roles.
```
Comment (rfroetscher) [inline docs/ADVANCED_CONFIGURATION.md]: it is lower case `admin`

File: docs/ADVANCED_CONFIGURATION.md [hunk 3]
```diff
@@ -1,18 +1,80 @@
...
+
+#### SAML Authorization
+If you are using SAML authorization you can specify additional configuration file `auth-attribute-map.yml`. This file will be ingested as is into `root_dir/config`
```
Comment (rfroetscher) [inline docs/ADVANCED_CONFIGURATION.md]: `auth-attribute-map.yml` is used for ingesting IDP provided user attributes into the user record. the most important of these is `role`  . they'll also need to configure `saml_issuer`, and `saml_name_identifier_format` in config.yml and have the `SAML_IDP_CERT`, `SAML_METADATA_URL`, and `SAML_SSO_TARGET` env variables

File: docs/ADVANCED_CONFIGURATION.md [hunk 4]
```diff
@@ -1,18 +1,80 @@
...
+
+## Alert Emails
+You can configure Aleph to email recipients about the status of Alerts. To do this you have to supply `smtp_settings` and a `default_url_host`.
```
Comment (rfroetscher) [inline docs/ADVANCED_CONFIGURATION.md]: also the `SMTP_PASSWORD` env variable

**Comment by rfroetscher** [comment]
looks good aside form those few edits :ear_of_rice:

**Comment by dollschasingmen** [comment]
@rfroetscher made some changes to reflect our new way of doing playground and added a gem executables doc. one more look?

File: docs/ADVANCED_CONFIGURATION.md [hunk 5]
```diff
@@ -1,18 +1,86 @@
 # Advanced Configuration
-Configuration explanations and examples can be found in the example [config directory](../config/example).
+Configuration examples can be found in the example [config directory](../config/example). The directory for your configurations is specified by the [environment variable](ENVIRONMENT_VARIABLES.md) `ALEPH_CONFIG_PATH`. If this is not set, Aleph will try to find them in `/tmp/aleph/configuration` by default.
+
+Configuration files do not contain secrets; those should be environment variables.
```
Comment (rfroetscher) [inline docs/ADVANCED_CONFIGURATION.md]: you can put them in the env.yml file, though. right?

File: docs/GEM_EXECUTABLES.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+##### Init Db
+Create the database by calling `rake db:create`.
+
+    aleph init_db -r RAILS_ENV
+
+ - `RAILS_ENV` will default to `development`.
+
+##### Init Db
+Uupdate the database by calling `rake db:migrate`.
```
Comment (rfroetscher) [inline docs/GEM_EXECUTABLES.md]: typo

**Comment by rfroetscher** [comment]
looks good aside from those two comments

:japanese_castle:

**Pull request merged**